main.rb

```diff
@@ -0,0 +1,21 @@
+class Person
```

Comment:
- [ ] It is a convention in Ruby (see [link](https://namingconvention.org/ruby/file-naming.html)) to name our class files the same name as the contained class as it helps us write readable code. Plus, at some point in a later project of this OOP, you will be REQUIRED to explicitly create a **`main.rb`** file. Kindly change the file name from **`main.rb`** to **`person.rb`**.